website/content/docs/getting-started.md

```diff
@@ -88,7 +95,8 @@ $ heron-tracker
 ... Using config file: /Users/USERNAME/.herontools/conf/heron_tracker.yaml
 ```
 
-You can reach Heron Tracker in your browser at http://localhost:8888.
+You can reach Heron Tracker in your browser at http://localhost:8888 and see something like this
```

Comment:
It's convenient to make these hrefs like [http://localhost:8888](http://localhost:8888). Same for below.